install/package-development.rst

```diff
@@ -2,7 +2,7 @@
 Building a package with the installed Science Pipelines stack
 #############################################################
 
-You can build packages alongside an LSST Science Pipelines stack that you have installed with :doc:`newinstall.sh <newinstall>`, :doc:`lsstsw <lsstsw>`, or :doc:`Docker <docker>`.
+You can build packages alongside an LSST Science Pipelines stack that you have installed with :doc:`lsstsw <lsstsw>` or :doc:`Docker <docker>`.
```

Comment:
Why leave lsstinstall out of this list and the one below?

Reply:
Indeed. Fixed.